### ISSUE TITLE
Add support for 32bit arm 

### ISSUE THREAD

**Issue by igor-microblink** [wx257osn2/qoixx#26]
At the moment 32bit android arm is not supported, so the fallback without SIMD is required.

**Comment by wx257osn2** [comment]
@igor-microblink Does [this change](https://github.com/wx257osn2/qoixx/commit/f5f9270a42d1ef25a486af9e3ab8119f956f4283) fix this? If so, I will merge this into `master` .

**Comment by igor-microblink** [comment]
Thanks. Proposed change is fix for compilation error on armebi v7a. 

__ARM_FEATURE_SVE is not defined for armeabi-v7a. Is it possible to add impl which enabled arm neon for 32bit ARMs? AFAIK ARM NEON is supported [from armeabi-v7](https://developer.arm.com/documentation/den0013/d/Introducing-NEON/NEON-architecture-overview/NEON-instruction-set).

**Comment by wx257osn2** [comment]
> __ARM_FEATURE_SVE is not defined for armeabi-v7a

Yes, cause Arm SVE is an extension for armv8 ~~and one of basic instructions in armv9~~ .

> Is it possible to add impl which utilizes arm neon for 32bit ARMs? AFAIK ARM NEON is supported [from armeabi-v7](https://developer.arm.com/documentation/den0013/d/Introducing-NEON/NEON-architecture-overview/NEON-instruction-set).

Hmm, currently qoixx doesn't support armv7 as SIMD-available environment, because I don't have any development environment of armv7. It is possible _technically_ cause NEON is an extension for armv7, but qoixx basically assumes 64-bit architecture (e.g. [fast hash calculation using 64bit integer arithmetic](https://github.com/wx257osn2/qoixx/blob/3c6666c8df599a179815e420f7f3d5e496e3de3d/include/qoixx.hpp#L235-L241); this might be not fast on 32bit environment), and currently I don't think I'd support it aggressively. Of course PR for supporting armv7 with NEON would be welcome, I will review it.
